README.md

```diff
@@ -64,6 +81,8 @@ If you use `MESH_LOWPAN_ND` or `MESH_THREAD` you need to specify your radio modu
     }
 ```
 
+### CELLULAR_ONBOARD
+
 If you use `CELLULAR_ONBOARD` (for which user documentation can be found [here](https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/APIs/communication/cellular/)) you must specify the following:
```

Comment:
Suggesting that you link like this: 
If you use` [`CELLULAR_ONBOARD`](https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/APIs/communication/cellular/)`